tests/testUnitVector3d.py

```diff
@@ -46,7 +51,7 @@ def testConstruction(self):
         self.assertAlmostEqual(u.x(), 0.5, places=15)
         self.assertAlmostEqual(u.y(), 0.5, places=15)
         self.assertAlmostEqual(u.z(), 0.5 * math.sqrt(2.0), places=15)
-        u = UnitVector3d.northFrom(u.asVector3d())
+        u = UnitVector3d.northFrom(u)
```

Comment:
This code would be clearer if used a new variable name each time instead of changing the value of `u` several times in the same method.